### ISSUE TITLE
rhandsontable not visible in RStudio viewer

### ISSUE THREAD

**Issue by helgasoft** [jrowen/rhandsontable#212]
Using the latest R and RStudio, command _rhandsontable(mtcars)_ does not show a table in the **Viewer tab**. Clicking icon "_Show in new window_" opens a browser tab with table displayed as expected.
FYI, _rhandsontable_ was displaying fine in R v.3.3.3.
```
> library("rhandsontable", lib.loc="~/R/win-library/3.4")
> rhandsontable(mtcars)

> sessionInfo()
R version 3.4.3 (2017-11-30)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.4.3 tools_3.4.3   
```
Edit:  the Javascript **console** is showing the following errors:
```
Unexpected CSS token: : handsontable.full.min.css:297
Invalid CSS property declaration at: * handsontable.full.min.css:34
`language` is deprecated since version 1.6.0. Use `culture` instead handsontable.full.min.js:36
SyntaxError: Expected token ')' rhandsontable.js:332
```

**Comment by jrowen** [comment]
Please try the dev version. We'll push an update to Cran shortly.  Thanks
for the excellent detail.

**Comment by peterfrescura** [comment]
Hi, 
I had the same issue and can confirm rolling back to 0.3.4 works on mint too.

`> library("rhandsontable")`
`> rhandsontable(mtcars)`
`> sessionInfo()`
`R version 3.4.3 (2017-11-30)`
`Platform: x86_64-pc-linux-gnu (64-bit)`
`Running under: Linux Mint 18.1`

`Matrix products: default`
`BLAS: /usr/lib/openblas-base/libblas.so.3`
`LAPACK: /usr/lib/libopenblasp-r0.2.18.so`

`locale:`
` [1] LC_CTYPE=en_ZA.UTF-8       LC_NUMERIC=C               LC_TIME=en_ZA.UTF-8 `
` [4] LC_COLLATE=en_ZA.UTF-8     LC_MONETARY=en_ZA.UTF-8    LC_MESSAGES=en_ZA.UTF-8` 
 `[7] LC_PAPER=en_ZA.UTF-8       LC_NAME=C                  LC_ADDRESS=C`
`[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_ZA.UTF-8 LC_IDENTIFICATION=C `

`attached base packages:`
`[1] stats     graphics  grDevices utils     datasets  methods   base`

`other attached packages:`
`[1] rhandsontable_0.3.4`

`loaded via a namespace (and not attached):`
`[1] htmlwidgets_0.9 compiler_3.4.3  magrittr_1.5    htmltools_0.3.6 tools_3.4.3`
`[6] yaml_2.1.16     Rcpp_0.12.14    jsonlite_1.5    digest_0.6.14`

**Comment by jrowen** [comment]
This has been fixed in the dev version.  Sorry for the inconvenience.

**Comment by helgasoft** [comment]
Confirm as fixed.

**Comment by dominic007** [comment]
Its still not rendering. Where can I get the dev version?

**Comment by jrowen** [comment]
You can install the dev version with the command below.
```
devtools::install_github("jrowen/rhandsontable")
```

**Comment by jrowen** [comment]
Fixed in v0.3.6.  Thanks for reporting.
